featurebyte/__init__.py

```diff
@@ -126,6 +126,26 @@ def get_active_profile() -> Profile:
         If no profile is found in configuration file
     """
     conf = Configurations()
+
+    # check if we are in DataBricks environment and valid secrets are present create a profile automatically
+    db_utils = globals().get("dbutils")
+    if db_utils:
+        databricks_auto_profile_name = "databricks-auto-profile"
+        if conf.default_profile_name != databricks_auto_profile_name:
+            try:
+                api_url = db_utils.secrets.get(scope="featurebyte", key="api-url")
+                api_token = db_utils.secrets.get(scope="featurebyte", key="api-token")
+                register_profile(
+                    profile_name=databricks_auto_profile_name,
+                    api_url=api_url,
+                    api_token=api_token,
+                    make_default=True,
```

Comment:
Indeed. But the config class will always create a local profile if the config file does not exist so there will be always at least one profile with the name "local".

Reply:
If the config file does exist, will the `databricks-auto-profile` still be created and made the default?

Reply:
the config file will be created before this block is reached, and the local profile will not work for databricks since the local service won't be available. Should we check if the default profile is accessible and create the `databricks-auto-profile` profile only if is not?

Reply:
The the local profile was automatically created, that is fine. I was mainly wondering about the other case: there is an existing profile created by user (and not the local profile created automatically by featurebyte).

Reply:
I guess there might not be a straightforward way of checking whether a profile was automatically created or manually created. I think the current approach should work for most cases. If we want to be a bit careful, maybe we can do something like only adding this new default profile if the original profile was named "local" (most likely that was created automatically by featurebyte).

Reply:
Got it - I will update the logic to check for existing profile other than the default local